internal/series/series.go

```diff
@@ -71,7 +72,7 @@ func ProcessUpdate(event event.Event, ps pointstores.PointStore, tServ chan type
 			}
 		}
 		// Queue up training if enough points are available
-		req, err := ml.Required(len(inputs), 1, conf.ML.HLayers, conf) // 1 because we'll be creating individual nets for each output
+		req, _ := ml.Required(len(inputs), 1, conf.ML.HLayers, conf) // 1 because we'll be creating individual nets for each output
```

Comment:
At the moment, the only reason for err to not be nil would be for an invalid net type to be configured. But it doesn't make sense to show an error about it this at this point in the execution, that is why I've opted to add a check in the config module and discard the value here instead.